[2024-02-21T07:38:07.340+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-31T00:00:00+00:00 [queued]>
[2024-02-21T07:38:07.348+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-31T00:00:00+00:00 [queued]>
[2024-02-21T07:38:07.348+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:38:07.358+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-01-31 00:00:00+00:00
[2024-02-21T07:38:07.368+0000] {standard_task_runner.py:60} INFO - Started process 106 to run task
[2024-02-21T07:38:07.374+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-01-31T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmprsxb5mz_']
[2024-02-21T07:38:07.379+0000] {standard_task_runner.py:88} INFO - Job 73: Subtask get_mail
[2024-02-21T07:38:07.459+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-01-31T00:00:00+00:00 [running]> on host 893865381b0d
[2024-02-21T07:38:07.603+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-31T00:00:00+00:00'
[2024-02-21T07:38:07.604+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 196, in execute
    context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 261, in context_merge
    context.update(*args, **kwargs)
  File "/usr/local/lib/python3.8/_collections_abc.py", line 837, in update
    for key, value in other:
ValueError: too many values to unpack (expected 2)
[2024-02-21T07:38:07.627+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240131T000000, start_date=20240221T073807, end_date=20240221T073807
[2024-02-21T07:38:07.671+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 73 for task get_mail (too many values to unpack (expected 2); 106)
[2024-02-21T07:38:07.712+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T07:38:07.733+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T07:59:38.671+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-31T00:00:00+00:00 [queued]>
[2024-02-21T07:59:38.686+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-31T00:00:00+00:00 [queued]>
[2024-02-21T07:59:38.687+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:59:38.704+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-01-31 00:00:00+00:00
[2024-02-21T07:59:38.712+0000] {standard_task_runner.py:60} INFO - Started process 369 to run task
[2024-02-21T07:59:38.715+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-01-31T00:00:00+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpex2emj_y']
[2024-02-21T07:59:38.719+0000] {standard_task_runner.py:88} INFO - Job 127: Subtask get_mail
[2024-02-21T07:59:38.774+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-01-31T00:00:00+00:00 [running]> on host 9bab6a0db245
[2024-02-21T07:59:38.857+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-31T00:00:00+00:00'
[2024-02-21T07:59:41.939+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T07:59:44.985+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T07:59:44.987+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-152.txt')
[2024-02-21T07:59:44.988+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.988+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T07:59:44.989+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-153.txt')
[2024-02-21T07:59:44.990+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.990+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T07:59:44.991+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-154.txt')
[2024-02-21T07:59:44.992+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.992+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T07:59:44.993+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-155.txt')
[2024-02-21T07:59:44.994+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.994+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T07:59:44.996+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-156.txt')
[2024-02-21T07:59:44.996+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.997+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T07:59:44.998+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-157.txt')
[2024-02-21T07:59:44.998+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.999+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T07:59:45.000+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-158.txt')
[2024-02-21T07:59:45.001+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.001+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T07:59:45.003+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-159.txt')
[2024-02-21T07:59:45.003+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.004+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T07:59:45.005+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-160.txt')
[2024-02-21T07:59:45.005+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.006+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T07:59:45.007+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-161.txt')
[2024-02-21T07:59:45.007+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.721+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T07:59:45.732+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240131T000000, start_date=20240221T075938, end_date=20240221T075945
[2024-02-21T07:59:45.785+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T07:59:45.800+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
